src/interpreter/Interpreter.ts

```diff
@@ -78,11 +79,11 @@ export class Interpreter {
     interpretItems(schema, model, this, interpreterOptions);
     interpretProperties(schema, model, this, interpreterOptions);
     interpretAllOf(schema, model, this, interpreterOptions);
+    interpretOneOf(schema, model, this, interpreterOptions);
     interpretDependencies(schema, model, this, interpreterOptions);
     interpretConst(schema, model);
     interpretEnum(schema, model);
 
-    this.interpretAndCombineMultipleSchemas(schema.oneOf, model, schema, interpreterOptions);
     this.interpretAndCombineMultipleSchemas(schema.anyOf, model, schema, interpreterOptions);
```

Comment:
Even though this PR is for `oneOf` the same behavior can be applied to `anyOf` as they have, from a data model perspective, the same expected behavior.

Dont need to adapt it, but just wanted to raise it 🙂

Reply:
I can do that in this MR :)

Reply:
Please check now